dashboard/lib/services/script_seed.rb

```diff
@@ -462,6 +468,27 @@ def self.import_lessons_vocabularies(lessons_vocabularies_data, seed_context)
       LessonsVocabulary.joins(:lesson).where('stages.script_id' => seed_context.script.id)
     end
 
+    def self.import_lessons_programming_expressions(lessons_programming_expressions_data, seed_context)
+      return [] if lessons_programming_expressions_data.blank?
+
+      lessons_programming_expressions_to_import = lessons_programming_expressions_data.map do |lv_data|
+        lesson_id = seed_context.lessons.select {|l| l.key == lv_data['seeding_key']['lesson.key']}.first&.id
+        raise 'No lesson found' if lesson_id.nil?
+
+        programming_expression_id = ProgrammingExpression.find_by(key: lv_data['seeding_key']['programming_expression.key']).id
```

Comment:
this is not good because it will do too many queries. instead, load all the ProgrammingExpressions into the seed context.